src/senaite/core/profiles/default/workflows/senaite_sample_workflow/definition.xml

```diff
@@ -1580,6 +1590,16 @@
     </guard>
   </transition>
 
+  <!-- Transition: Multi Results -->
+  <transition transition_id="listing_multi_results" title="Multi Results" new_state="" trigger="USER"
+              allowed="False" before_script="" after_script="" i18n:attributes="title">
+    <action url="" category="workflow" icon="">Multi Results</action>
+    <guard>
+      <guard-expression>python:here.guard_handler("restore")</guard-expression>
```

Comment:
Misspelling: "restore" -> "listing_multi_results"

Also consider to add a `guard_listing_multi_results`  guard at https://github.com/senaite/senaite.core/blob/2.x/src/bika/lims/workflow/analysisrequest/guards.py that would perform the necessary checks below (see comment re transition in `sample_due` status):

1) remaining analyses not yet submitted
2) "field" / "lab" analyses

Reply:
I intentionally did not add a guard with checks because I believe it is not necessary, because actually nothing can happen when analyses are submitted. Then the analyses listings render the results like when you view the sample itself.
Therefore, I would rather stick to a permission than to complicated guards that have a negative impact on the performance.

Reply:
Ok, noted!